app/Commands/DailyReviewCommand.php

```
@@ -0,0 +1,502 @@
+        ]);
+    }
+
+    /**
+     * @param  array<string, string>  $reflections
+     * @param array{
+     *     knowledge: Collection<int, Entry>,
+     *     prs: array<int, array<string, mixed>>,
+     *     issues: array<int, array<string, mixed>>
+     * } $milestones
+     */
+    private function formatContent(array $reflections, array $milestones, bool $isQuick): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$isQuick'. (undefined)

(UnusedFormalParameter)